clients/frontend-authorization/src/oidc/browser/BrowserAuthorizationLogger.ts

```diff
@@ -55,7 +55,7 @@ export class BrowserAuthorizationLogger implements IOidcClientLogger {
   }
 
   /** Initializes forwarding of OidcClient logs to the Bentley Logger */
-  public static initializeLogger() {
+  public static initializeLogger(): void {
```

Comment:
@calebmshafer do you object to being explicit about return type in general or only to including such changes as part of this specific PR?

Reply:
`@typescript-eslint/explicit-module-boundary-types` requires this. It's now enabled by default and I didn't want to disable it because it could help spot errors. Unfortunately there's no way to whitelist `void` (it would make the rule very slow and complicated as it would have to infer return types) so for now I disabled it in the packages that had most of those warnings.

Reply:
@pmconne I was initially objecting to us requiring `void`, or `Promise<void>`, be specified as a return type and instead leaving those implied.  However, I do want us to be explicit about all other types so if there's no way to just skip those ones then this works for me.

 @paulius-valiunas okay, I did see that in a handful of `package.json`'s.  We should let each package owner know and have them take a pass at cleaning up the code.

Reply:
it really bugs me too, there's no reason to require `void` specified explicitly, but on the other hand I did find a few places in our public-facing code where the return type was hard to infer even when looking at the full Typescript source, not to mention just looking at type definitions or plain Javascript. I think this is a reasonable price to pay for more solid type checking

Reply:
I object to having to decorate every method return with a type when it's obvious. Is that what this rule requires?

Reply:
@kabentley the rule is not smart enough to figure out which methods' return types are obvious, so yes, it requires every method to have a return type defined, no matter how obvious it is. I'd prefer to set it to "warn" and keep it only as a gentle reminder to the developer to think about it (especially since we have squiggles in VSCode), but most of our packages have switched to a zero warnings policy so the only options are disabling the rule or enforcing it as an error.

Reply:
in that case, I suggest we turn it off. The compiler doesn't need the decorations, so it won't find real problems. The "hard to infer" is relative, and simply floating your cursor over it removes any doubt. If there were a rule that required it in any function with more than one return statement, that might be useful, but causing everyone to do useless typing is not helpful.

Reply:
I agree that we should change all warnings to errors, that's fine.

But, I don't *want* people adding ugly unnecessary type decorations where there're just clutter. I don't want this rule generating warnings because I don't think adding them is the right thing to do.

Reply:
OK fair enough. I'll disable the rule for now and see if I can find a way to extend it so it ignores "void" functions at least in 95% or so of our cases.

Reply:
No, I don't want it warning even for non-void methods. If you could find a way to make it warn ONLY if there is more than one return statement, then that would be useful. Otherwise, just turn it off.